task5_valid_parentheses.py

```diff
@@ -0,0 +1,24 @@
+def isValid(s):
+    """
+    Check if the string of parenthesis is valid.
+    :type s: str
+    :rtype: bool
+    """
+
+    if len(s) < 2:
+        return False
+
+    valid_check = [s[0]]
```

Comment:
the variable name seems to be not self-explanatory. please, try to use self-explanatory names, as this makes understanding code much easier. e.g:
```suggestion
    opened_brackets = [s[0]]
```